src/compiler/commandLineParser.ts

```diff
@@ -262,7 +262,9 @@ namespace ts {
                 "es3": ScriptTarget.ES3,
                 "es5": ScriptTarget.ES5,
                 "es6": ScriptTarget.ES6,
+                "es8": ScriptTarget.ES8,

```

Comment:
Officially there is no ES8, it is ES2017, so i would not add ES8 here or anywhere else. i would just leave it to ES2017